LOCAL.md

```diff
@@ -0,0 +1,48 @@
+# Local development
```

Comment:
I would move this to `docs/developer.md` - which already has a section about "developer setup".